client/inbox-panel/index.scss

```diff
@@ -34,3 +34,46 @@
 	transform: translateX(50%);
 	transition: opacity 500ms, transform 500ms, max-height 500ms;
 }
+
+.wooocommerce-inbox-card__header {
+	.woocommerce-badge {
+		margin-left: 16px;
+	}
+}
+
+.woocommerce-inbox-dismiss-all-modal {
+	width: 565px;
+	max-width: 100%;
+
+	.components-modal__header {
+		border-bottom: 1px solid #ddd !important;
+	}
+
+	.woocommerce-usage-modal__message {
+		box-sizing: border-box;
+		border-bottom: 1px solid #ddd;
+		padding: $gap * 2;
```

Comment:
The padding (or maybe elements around it) are a little funky with the latest GB installed.  Might need to add another style here to cover both cases (GB installed and not).
![Screen Shot 2021-11-05 at 3 58 58 PM](https://user-images.githubusercontent.com/10561050/140571366-08cf0c45-368b-4eb7-855e-c7fa013c76e3.png)

Reply:
Looks like the padding is coming from `.components-modal__content:before` so we can either remove that or reduce padding on this element.

Reply:
Thank you for catching it @joshuatf 👍  I totally missed it. Updated it in https://github.com/woocommerce/woocommerce-admin/pull/7871/commits/96eb962c5390eccb3a09f6a93ba2b6fbce6d4608